### PR TITLE
Support Jekyll app builds for static web apps

### PR DESCRIPTION
**Pull request by qianz2**
- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [x] Tests are included and/or updated for code changes.
- [x] Proper license headers are included in each file.

### CODE REVIEW

**Review by arroyc — COMMENTED** [review]
We basically want to cover following things ...
1. jekill app with app_type variable passed as static-sites
2. jekill app with app_type variable not passed
3. ruby app as app_type variable passed as static-sites
4. ruby app as app_type variable not passed

among these 4 there will be 2 negative tests I believe, we need to ensure we (oryx) and static site team agree to these, that's why we need at-least these 4 tests

File: src/Detector/Ruby/RubyPlatformDetectorResult.cs [hunk 1]
```diff
@@ -19,5 +19,10 @@ public class RubyPlatformDetectorResult : PlatformDetectorResult
         /// Gets or sets the value if a bundler version specified in 'Gemfile.lock' in the repo.
         /// </summary>
         public string BundlerVersion { get; set; }
+
+        /// <summary>
+        /// Gets or sets the value if a bundler version specified in 'Gemfile.lock' in the repo.
```
Comment (gpcastro) [inline src/Detector/Ruby/RubyPlatformDetectorResult.cs:24]: Is this correct?

**Review by gpcastro — APPROVED** [review]

**Review by arroyc — APPROVED** [review]

**Comment by arroyc** [comment]
its best to merge after we produce tag for ant-93

**Review by arroyc — CHANGES_REQUESTED** [review]
Stella lets discuss the test scenarios with anthony tomorrow .. and then we can finalize that here

File: src/Detector/Ruby/RubyPlatformDetectorResult.cs (hunk 1 above)
Comment (qianz2) [reply src/Detector/Ruby/RubyPlatformDetectorResult.cs:24]: It's not, thanks for catching it, will update

**Review by arroyc — APPROVED** [review]

**Pull request merged**